board/safety/safety_subaru.h

```diff
@@ -161,7 +203,11 @@ static int subaru_fwd_hook(int bus_num, int addr) {
   int bus_fwd = -1;
 
   if (bus_num == SUBARU_MAIN_BUS) {
-    bus_fwd = SUBARU_CAM_BUS;  // forward to camera
+    // Global Platform
+    bool block_msg = subaru_longitudinal && ((addr == MSG_SUBARU_Brake_Status) || (addr == MSG_SUBARU_CruiseControl));
+    if (!block_msg) {
+      bus_fwd = SUBARU_CAM_BUS;  // to the eyesight camera
+    }
```

Comment:
What does this do? Why block these messages to the camera? Comment explaining this would be a good idea

Reply:
Brake_Status and CruiseControl are intercepted and rewritten in openpilot.

Brake_Status is to fake the ES_Brake feedback signal to eyesight, which was for preserving AEB.
CruiseControl is to tell eyesight that it is not enabled (we set Cruise_Activated to false), so it only will try and perform AEB

Reply:
Why can't you let the EyeSight camera see the car's signals? Will it thinking it's in control of ACC somehow change the AEB behavior?

@adeebshihadeh told me that when working on the first iteration of an experimental longitudinal support PR, we don't need or want to take the time to verify that AEB still works, since we don't guarantee that, and when we do ship this to release (not master), we will need to re-do AEB testing anyway.

If there are no visible faults, I think we should remove this blocking/forwarding (complexity) and just show the AEB disabled light for now.